crates/emitter/src/emitter.rs

```diff
@@ -681,6 +681,40 @@ impl Emitter {
             FunctionItem::Statement(x) => self.statement(&x.statement),
         };
     }
+
+    fn default_enum_type(&mut self, arg: &EnumDeclaration) {
+        let enum_symbol = symbol_table::resolve(arg.identifier.as_ref())
+            .unwrap()
```

Comment:
This `unwrap` may be troublesome.
If there is any error around symbol table, this resolve may be failed.
So Veryl compiler may panic before reporting these errors.